<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="gbSerialPortInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>串口配置</string>
    </property>
    <widget class="QComboBox" name="cbSerialPortName">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblSerialPortName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>名称</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbSerialPortBaudRate">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblSerialPortBaudRate">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>波特率</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbSerialPortDataBits">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblSerialPortDataBit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>数据位</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbSerialPortStopBits">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblSerialPortStopBit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>停止位</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbSerialPortParity">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblSerialPortCheck">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>奇偶校验</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblSerialPortFlowControl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>流控制</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbSerialPortFlowControl">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>180</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbCalibration">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>211</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>校准值</string>
    </property>
    <widget class="QLabel" name="lblCaliSoundSpeed">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>声速</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCaliSoundSpeed">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCaliTemperature">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblCaliTemperature">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>温度</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCaliPressure">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblCaliPressure">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>压力</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCaliQuat1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblCaliQuat1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>姿态四元数1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCaliQuat2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>150</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblCaliQuat2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>姿态四元数2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCaliQuat3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>180</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblCaliQuat3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>姿态四元数3</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCaliQuat4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>210</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblCaliQuat4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>姿态四元数4</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCaliVertSpeed">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>240</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblCaliVertSpeed">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>垂直速度</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblCaliRefVoltage">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>基准电压</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCaliVoltage">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>270</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCaliReset">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>320</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>重置(&amp;R)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCaliVerify">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>更改(&amp;V)</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbDataCollection">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>751</width>
      <height>621</height>
     </rect>
    </property>
    <property name="title">
     <string>数据采集</string>
    </property>
    <widget class="QGroupBox" name="gbCurData">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>520</y>
       <width>731</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLineEdit" name="txtDataSoundSpeed">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lblDataSoundSpeed">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>声速</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtDataTemperature">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lblDataTemperature">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>30</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>温度</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtDataPressure">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>30</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lblDataPressure">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>30</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>压力</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtDataQuat1">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lblDataQuat1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>姿态四元数1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtDataQuat2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>60</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lblDataQuat2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>姿态四元数2</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtDataQuat3">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>60</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lblDataQuat3">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>60</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>姿态四元数3</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtDataQuat4">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>60</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lblDataQuat4">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>60</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>姿态四元数4</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtDataVertSpeed">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>30</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lblDataVertSpeed">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>30</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>垂直速度</string>
      </property>
     </widget>
    </widget>
    <widget class="QChartView" name="chart" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>731</width>
       <height>491</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnSerialPortConnect">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>连接(&amp;C)</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSerialPortState">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
